lib/auth/auth.go

```diff
@@ -1000,6 +1000,8 @@ type certRequest struct {
 	connectionDiagnosticID string
 	// attestationStatement is an attestation statement associated with the given public key.
 	attestationStatement *keys.AttestationStatement
+	// skipAttestation is a server-side flag which is used to skip the attestation check.
+	skipAttestation bool
```

Comment:
Where this logic for handling the `skipAttestation` flag  is implemented ?

Reply:
You're right, thanks for catching that. I guess I rushed it and didn't push the final change after testing it. Fixed now.